app/assets/javascripts/admin/addon/templates/config-about.hbs

```diff
@@ -1 +1,15 @@
-<AdminConfigAreas::About @data={{this.model.site_settings}} />
\ No newline at end of file
+<AdminPageHeader
+  @titleLabel="admin.config_areas.about.header"
```

Comment:
Should we have a `descriptionLabel` too? Also I vaguely remember talk of a Meta topic for this, if that's not ready yet fine but later on we should add a `learnMoreUrl` when it's written

Reply:
Agreed! A description and a learn more would be great here.